alerting/dashboardunifie/README.md

```diff
@@ -0,0 +1,85 @@
+# ALERTING Dashboard Unifie SignalFx integrations
+
+## How to use this module
+
+```hcl
+module "signalfx-integrations-alerting-dashboardunifie" {
+  source  = "github.com/claranet/terraform-signalfx-integrations.git//alerting/dashboardunifie"
+
+  dashboardunifie_url          = var.dashboardunifie_url
+  header_authorization_base64  = base64encode("username:${data.vault_generic_secret.dashboardunifie.data["token"]}")
```

Comment:
why not using base64encode function inside your module to ease the import for the user?

Reply:
Yes, i pushed a change so this is managed on the module side & updated the readme